astroquery/mast/core.py

```diff
@@ -1665,47 +1249,47 @@ def _download_files(self, products, base_dir, cache=True):
         response : `~astropy.table.Table`
         """
 
-        manifestArray = []
-        for dataProduct in products:
+        manifest_array = []
+        for data_product in products:
 
-            localPath = base_dir + "/" + dataProduct['obs_collection'] + "/" + dataProduct['obs_id']
-            dataUrl = self._MAST_DOWNLOAD_URL + "?uri=" + dataProduct["dataURI"]
+            local_path = base_dir + "/" + data_product['obs_collection'] + "/" + data_product['obs_id']
```

Comment:
I wonder how this worked on windows before, please use `os.path.join()` instead.